lib/hydra/derivatives.rb

```diff
@@ -13,6 +13,7 @@ module Derivatives
     autoload_under 'runners' do
       autoload :AudioDerivatives
       autoload :DocumentDerivatives
+      autoload :FullTextExtract
```

Comment:
I have a minor preference for ending in a noun here (`FullTextExtraction`) but I wouldn't hold the PR up over that alone.